admin/system/news-save.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    include_once("connection.php");
+
+    $judul = $_POST['judul'];
```

Comment:
Codacy found an issue: [Direct use of $_POST Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)